documentation/Database-Schema-Review.md

```diff
@@ -0,0 +1,99 @@
+
+The `time_created` is the time when the review has started. The created is when the first section is assigned to a reviewer.
+
+The `trigger` is updated everytime the review has changes done by users or a scheduler. See more about [triggers](Triggers-and-Actions.md).
+
+**To be considered:**
+
+- The `review` is per pair of **application-stage** or one per each time it goes back to the Applicant? I think it would be always one per each pair, although the field `review.comment` implies one overall comment which should be sent to the Applicant - per review. In the case if the review is re-submitted and another review will be started (on the same stage), how we would we keep this comment? Or do we remove it or do we create another `review`?
+- The `status` of `'Ready'` - which is when a lower level reviewer has finished the review and the consolidation can happen - is defined in the Schema latest documentation of Drive as `'Reviewed'`. Is it what we have agreed on? I can update it, just want to check that first.
+
+### review section join
+
+The `review_id` links to the main review record of the **application-stage** pair.
+
+The `review_section_assignment_id` links to the reviewer of the section.
+
+The `review_section_id` links to the section being reviewed.
+
+The `send_to_applicant` is a flag for adding this section (and `review_section.comment`) to be sent back to the Applicant.
+
+**To be considered:**
+
+- Can we change `review_section_assignment` to `review_assignment`. Just to keep it short. We know that there will be 1 section per 1 review assignment, the table don't need to have such a long name. And when refereced in `review_section_join`
+  the `section` is duplicated in `review_section_join.review_section_assignment_id`.
+
+- The field `section` linking to the `Application_section` in both `review_section_assignment` and `review_sectiob_join`. One has to go.
+
+### review section
+
+The `decision` is either: `'Approved'`, `'Rejected'`, `'Observations'` and what is the overall decision per section. It is automatically defined by the combinatiion of `review_response.decision`. See [SQL functions](Functions.md) for more details.
+
+The `comment` is also an overall comment of the whole section being reviewed. It can be sent to the Applicant with the review.
+
+### review section assignment
+
+The `reviewer_id` links to the user assigned to review a section of the application.
+
+The `assigner_id` links to the user assigner of the Reviewer.
+
+The `stage_id` links to the review's stage of the application.
+
+The `section_id` links to the review's section of the application.
+
+The `level` is defined by the permission of the reviewer associated with this review. Added to here to facilitate finding out which reviews have been done.
+
+**To be considered:**
+
+- The name `review_section_assginment` is quite long and doesn't add much. Since the entity already stores the `section_id` should we just rename it to `review_assignement`?
```

Comment:
Yeah, I guess that makes sense. Slightly concerned it might create confusion but I can see the advantage too. Maybe see what @andreievg  thinks?